src/main/inAppNotifications/views/InAppNotificationBaseView.tsx

```
@@ -0,0 +1,157 @@
+  resource: {
+    url: string;
+  };
+  contributor?: {
+    avatarUrl: string;
+  };
+  triggeredAt: Date;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  values: any;
+}
+
+const Wrapper = <D extends React.ElementType = ListItemButtonTypeMap['defaultComponent'], P = {}>(
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)